nncf/onnx/graph/node_utils.py

```diff
@@ -0,0 +1,55 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import onnx
+import numpy as np
+
+from nncf.common.graph.graph import NNCFNode
+from nncf.onnx.graph.metatypes.onnx_metatypes import LAYERS_WITH_BIAS_METATYPES
+from nncf.onnx.graph.onnx_graph import ONNXGraph
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNX_OPERATION_METATYPES
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXIdentityMetatype
+
+
+def is_node_with_bias(node: NNCFNode) -> bool:
+    """
+    Checks if a bias is applied to the node's operation or not.
+
+    :param node: The node to check.
+    :return: Return `True` if `node` corresponds to the operation
+        with bias (bias is added to the output tensor of that operation),
+        `False` otherwise.
+    """
+    input_tensor_names = node.layer_attributes.input_tensor_names
+    return node.metatype in LAYERS_WITH_BIAS_METATYPES and len(input_tensor_names) > 2
+
+
+def get_bias_value(node: NNCFNode, model: onnx.ModelProto) -> np.ndarray:
+    """
+    Returns the bias value that is applied to the output tensor of the node's operation.
```

Comment:
```suggestion
    Returns the bias tensor of the node.
```

Reply:
Could you please explain what is the reason for this suggestion?

Reply:
Your descriptions confuses. What you mean by applied? Who applies? Does this bias somehow connected to initial node? I suggest to keep it simple

Reply:
Done